lib/membrane_camera_capture_plugin/camera.ex

```diff
@@ -82,4 +84,14 @@ defmodule Membrane.CameraCapture do
   def handle_info({:frame_provider, _buffer}, _ctx, state) do
     {[], state}
   end
+
+  defp pixel_format_to_atom("yuv420"), do: :I420
+  defp pixel_format_to_atom("yuv422p"), do: :I422
+  defp pixel_format_to_atom("yuv444p"), do: :I444
+  defp pixel_format_to_atom("rgb24"), do: :RGB
+  defp pixel_format_to_atom("rgba"), do: :RGBA
+  defp pixel_format_to_atom("yuyv422"), do: :YUYV
```

Comment:
We're using a [GStreamer naming convention for pixel formats](https://gstreamer.freedesktop.org/documentation/video/video-format.html?gi-language=c#enumerations). Cross-referencing with [ffmpeg's list](https://ffmpeg.org/doxygen/4.1/pixfmt_8h.html#a9a8e335cf3be472042bc9f0cf80cd4c5),  I think it should be `YUY2`